src/main/java/com/artgallery/domain/model/BankAccount.java

```diff
@@ -13,6 +13,7 @@ public class BankAccount {
     private String name;
     private String accountNumber;
     private BigDecimal balance;
+    private Currency currency;
 
     public BankAccount() {
```

Comment:
@NoArgsConstructor jeżeli w ogóle jest uzywany konstruktor bezparametrowy

Reply:
świecił się na żółto tak jakby był, to dodałam :)